vj4/handler/record.py

```diff
@@ -9,6 +9,7 @@
 from vj4 import constant
 from vj4 import error
 from vj4.handler import base
+from vj4.handler import contest as contestHandler
```

Comment:
Bad name `contestHandler`

Reply:
Could you suggest a better name?

Reply:
Just `contest` if no other `contest`s

Reply:
it is confusing for others. since in the rest of code contest is referring to model.contest.